pom.xml

```diff
@@ -171,6 +210,12 @@
             <version>${test.containers.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>elasticsearch-rest-client</artifactId>
```

Comment:
check whether we can use the existing JestClient to perform the needful operations instead of adding a new dependency.